packages/react-native-web/src/exports/ScrollView/index.js

```diff
@@ -17,21 +17,21 @@ import ScrollResponder from '../../modules/ScrollResponder';
 import ScrollViewBase from './ScrollViewBase';
 import StyleSheet from '../StyleSheet';
 import View from '../View';
-import React from 'react';
+import * as React from 'react';
 
-type ScrollViewProps = {
-  ...ViewProps,
+type ScrollViewProps = {|
+  ...React.ElementConfig<typeof ScrollViewBase>,
```

Comment:
Types should be imported separately rather than assuming they are available on the `React` object